nativelib/src/main/scala/scala/scalanative/native/alwaysinline.scala

```diff
@@ -0,0 +1,5 @@
+package scala.scalanative
+package native
+
+/** Unconditionally inline annotated method. */
+final class alwaysinline extends scala.annotation.StaticAnnotation
```

Comment:
How is that different from the standard `@inline`? In Scala.js and in the 2.12 JVM optimizer, `@inline` basically means "always inline" (if at all possible, obviously; for example if the call is polymorphic there's no way to do it).